lib/device.js

```diff
@@ -98,6 +104,17 @@ function device(_system, panel) {
 		}
 	};
 
+	self.timeouttimer = setInterval(function(){
+		if(self.userconfig.pin_timeout != 0 && self.userconfig.pin_enable == true && Date.now() >= self.lockoutat && !self.lockedout){
+			self.lockedout = true;
+			self.drawPage();
+		}
+		if(self.lockedout && !self.userconfig.pin_enable){
+			self.lockedout = false;
+			self.drawPage();
+		}
+	},1000);
```

Comment:
Really nitpicking, but space before 1000, as you actually usually have spaces after each argument, but not right here. And it is much prettier with space :)